packages/babel-core/src/api/browser.js

```diff
@@ -1,112 +0,0 @@
-/* eslint max-len: 0 */
```

Comment:
Oh! I was totally wrong about the `package.json` point I made, sorry. I guess the code I was thinking of got removed a while ago.

Yeah, my proposal was to move `api/node.js` to `index.js` since that is one of the few places that loads it. I'd expect you to have to update a couple other references to load the index instead, but that seems fine?